jax/_src/test_util.py

```diff
@@ -773,6 +773,33 @@ def rand_uniform(rng, low=0.0, high=1.0):
   return partial(_rand_dtype, rng.rand, post=post)
 
 
+def rand_fresnel(rng):
```

Comment:
I don't think we should add a new test utility for a single test.

Why not use `rand_default` with `scale=10000` or something like that?

Reply:
Good point. This was an easy way for me to make sure to sample points in the three regions, but I can use `rand_default` for sure.